app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -333,6 +334,12 @@ class Config {
     return GithubService(github);
   }
 
+  Future<FlutterDestination> createMetricsDestination() async {
```

Comment:
can we add a doc briefly explain Metrics/flutter destination?

Reply:
Done.